pmd-core/src/main/java/net/sourceforge/pmd/renderers/CodeClimateRule.java

```diff
@@ -53,45 +47,4 @@ public String toString() {
             return result;
         }
     }
-
```

Comment:
CodeClimateRule doesn't appear to be used anymore, not even by the renderer. So, the interface can probably be removed entirely and the CodeClimateCategory enum be moved into the CodeClimateRenderer?